syntax = "proto3";

package iroha.protocol;

import "block.proto";
import "google/protobuf/empty.proto";

message ToriiResponse {}
message QueryResponse {}
message QueueTransactionResponse {}

service CommandService {
  rpc Torii (Transaction) returns (ToriiResponse);
}

service QueryService {
//  rpc Find (QueryRequest) returns (QueryResponse);
}

service SumeragiService {
  rpc Verify (Block) returns (google.protobuf.Empty);
}

service OrderingService {
  rpc QueueTransaction (Transaction) returns (google.protobuf.Empty);
}
